### ISSUE TITLE
 'dict' object has no attribute 'username'

### ISSUE THREAD

**Issue by stallonen** [archlinux/archinstall#1792]
I tried to install without setting up a user and that seems to have caused an error. Do I always have to setup an user?
Here's the traceback 

```
Traceback (most recent call last):
  File "/usr/bin/archinstall", line 8, in <module>
    sys.exit(run_as_a_module())
             ^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/archinstall/__init__.py", line 266, in run_as_a_module
    importlib.import_module(mod_name)
  File "/usr/lib/python3.11/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1206, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1178, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1149, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 690, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/usr/lib/python3.11/site-packages/archinstall/scripts/guided.py", line 276, in <module>
    perform_installation(archinstall.storage.get('MOUNT_POINT', Path('/mnt')))
  File "/usr/lib/python3.11/site-packages/archinstall/scripts/guided.py", line 135, in perform_installation
    with Installer(
  File "/usr/lib/python3.11/site-packages/archinstall/lib/installer.py", line 137, in __exit__
    raise exc_val
  File "/usr/lib/python3.11/site-packages/archinstall/scripts/guided.py", line 195, in perform_installation
    PipewireProfile().install(installation)
  File "/usr/lib/python3.11/site-packages/archinstall/default_profiles/applications/pipewire.py", line 40, in install
    self._enable_pipewire_for_all(install_session)
  File "/usr/lib/python3.11/site-packages/archinstall/default_profiles/applications/pipewire.py", line 35, in _enable_pipewire_for_all
    install_session.arch_chroot('systemctl enable --user pipewire-pulse.service', run_as=user.username)
                                                                                         ^^^^^^^^^^^^^
AttributeError: 'dict' object has no attribute 'username'
```

Here is the [logfile](https://bpa.st/EGEAG)

**Comment by svartkanin** [comment]
Let me try to reproduce this

**Comment by svartkanin** [comment]
Okay I just run it with Python 3.11 and the same config as you had but it went through fine. 
Could you export the configuration for the user setting `user_credentials.json` and share that PLEASE REPLACE USERNAME AND PASSWORD with `dummy`, I'm just interested in the structure of the file.

**Comment by stallonen** [comment]
Can you tell me where might I find `user_credentials.json`?

**Comment by svartkanin** [comment]
ehm that's empty though, did you not set any users or root password?

**Comment by stallonen** [comment]
I set a root password and I didn't set up any users.

**Comment by svartkanin** [comment]
ahh okay that makes sense, then I think I know what's wrong

**Comment by stallonen** [comment]
PR 1794 seems to fix issue 1792? I just ran an install with 1794 applied locally to the master branch, and there were no issues whilst not configuring an user.

**Comment by svartkanin** [comment]
I don't think that PR #1794 fixes this one, but PR #1795 should
